src/components/ColumnLeft/PrerequisitesManager/PrerequisitesManager.js

```diff
@@ -0,0 +1,59 @@
+export default function PrerequisitesManager({
+  activeConnections,
+  onSelectedVersion,
+  selectedVersion,
+  onSelectedConnection,
+  selectedConnection,
+  isLockedForm,
+}) {
+  return (
+    <div className="row">
+      <div className="col-md-6">
+        <div className="form-group">
+          <label htmlFor="connection">Connection:</label>
+          <select
+            name="connection"
+            className="form-control"
+            id="connection"
+            onChange={onSelectedConnection}
+            value={selectedConnection}
+            disabled={isLockedForm}
+          >
+            <option value="" disabled>
+              - Select -
+            </option>
+            {activeConnections.map((connection) => {
+              return (
+                <option
+                  key={connection.connection_id}
+                  value={connection.connection_id}
+                >
+                  {connection.their_label}
+                </option>
+              )
+            })}
+          </select>
+          {selectedConnection && (
```

Comment:
if each dropdown were in its own `<div className="form-group">` rather than a single div, it makes the spacing between them nicer, basically adding on this line:
```
        </div>
        <div className="form-group">
```